admiral/pkg/controller/common/common.go

```diff
@@ -702,3 +702,24 @@ func GetMeshPortsHelper(meshPorts string, destService *k8sV1.Service, clusterNam
 	}
 	return ports
 }
+
+func GenerateUniqueNameForVS(originNamespace string, vsName string) string {
+
+	if originNamespace == "" && vsName == "" {
```

Comment:
nitpick, you could just make them individual `if` condition rather than `else if`